monkestation/code/modules/clothing/head/costume.dm

```diff
@@ -129,3 +129,7 @@ BUNNY EARS
 /*
 END OF BUNNY EARS
 */
+
+/obj/item/clothing/head/costume/bunnysuit/regular //the real bunny hat
```

Comment:
should change this to `/obj/item/clothing/head/costume/bunnyhead/regular` instead so the linters stop screaming. atleast if the comment is true, bunny suit is defined below